src/server/server.py

```diff
@@ -135,13 +138,15 @@ def create(
         assert (block_indices is None) != (num_blocks is None), "please specify num_blocks or block_indices, not both"
         if expiration is None:
             expiration = max(2 * update_period, MAX_DHT_TIME_DISCREPANCY_SECONDS)
+        if inference_max_length is None:
+            inference_max_length = max_batch_size
```

Comment:
Why do we assign max batch size to max sequence size?

Reply:
both are (meant to be) in tokens

Would you prefer to set it to a constant by default?